integ/failure-server/run.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
```

Comment:
I am essentially ignoring the implementation of this integration test as part of this review, but would love to see it be part of our CI workflow. This can be done by having an `integ` feature which enables integration tests, or it might be possible to run an otherwise-ignored test by name.

Reply:
> I am essentially ignoring the implementation of this integration test as part of this review, but would love to see it be part of our CI workflow. This can be done by having an `integ` feature which enables integration tests, or it might be possible to run an otherwise-ignored test by name.

I am going to experimentally push a version with the test enabled because I can't tell from searching whether or not docker already exists on the GitHub runner host. If it's there, it should 'just work'.

Reply:
Well, Docker is there, but for some reason the test didn't work. I added #163 to track enabling this test in CI.

Reply:
It works! [aab0f32](https://github.com/awslabs/tough/compare/2a161fa..aab0f32)